facial-emotion-classifier/node.html

```diff
@@ -0,0 +1,201 @@
+<script type="text/javascript">
+    RED.nodes.registerType('facial-emotion-classifier', {
+        category: 'Model Asset eXchange',
+        color: '#0bc6c0',
+        defaults: {
+            service: { value: "", type: "facial-emotion-classifier-service", required: true },
+            method: { value: "predict", required: true },
+
+            predict_image: { value: "" },
+            predict_imageType: { value: "str" },
+
+            name: { value: "" }
+        },
+        inputs: 1,
+        outputs: 1,
+        icon: "max-logo.png",
```

Comment:
`max-logo.svg`?

Reply:
Yep, just confirmed. Looks fine with the change

Reply:
thanks! That was it.